documentation/sdn/ovn_bgp_evpn.md

```diff
@@ -0,0 +1,81 @@
+# Using OVN BGP Agent with Evpn driver in Openstack Kolla
+>Note: This document is a draft
+
+This article covers the basics of BGP and EVPN and their applications. It focuses on the advantages of using OVN with the OVN-BGP agent compared to plain OVN. Additionally, it provides a demo setup of the OVN-BGP agent with EVPN in Kolla Ansible.
+
+## BGP
+Border Gateway Protocol (BGP) is the protocol used to exchange routing information between different networks. It's a key component that helps the Internet function by determining the best paths for data to travel from one point to another.
+
+## OVN BGP AGENT
+
+The OVN BGP Agent is a component designed to integrate the Open Virtual Network (OVN) with the Border Gateway Protocol (BGP) for routing purposes. OVN is a network virtualization system that provides software-defined networking (SDN) capabilities for virtualized environments, such as OpenStack. BGP, on the other hand, is the protocol used to exchange routing information between different networks (as explained in the previous message).
+
+### Advantages of OVN BGP vs pure OVN networking
+- **Direct External Network Reachability / connectivity** - External access to VMs and  connectivity of VMs to external network  is simplified because the BGP agent advertises their internal IP addresses without needing NAT, Floating IP or external gateway.
```

Comment:
```suggestion
- **Direct external network reachability / connectivity** - External access to VMs and  connectivity of VMs to external network  is simplified because the BGP agent advertises their internal IP addresses without needing NAT, Floating IP or external gateway.
```